docs/user-guide/settings/advanced.md

```diff
@@ -1,92 +1,161 @@
 # Advanced Settings #
 
-## Options ##
+This page enables advanced control of PhotoPrism,
+and configuring image and thumbnail handling.
 
-#### Debug Logs ####
+![](img/advanced-settings.jpg)
+
+The same options can all be controlled instead using their corresponding
+[config options](/getting-started/config-options/).
+Settings updated here will override values already configured.
```

Comment:
>  Settings updated here will override values already configured.

Is this true? I assumed the options set on this page take priority over any environment variables or command flags.

Reply:
These values will be stored in the `config` folder and override whatever was configured before e.g. via environment variables. Otherwise, making changes wouldn't make much sense.